docs/en/week10/10-1.md

```diff
@@ -0,0 +1,271 @@
+
+<div class="MathJax_Display" style="text-align: center;">
+$$
+\begin{aligned}
+    f_{\theta}(I) = f_{\theta}(\text{augment}(I))
+\end{aligned}
+$$
+</div>
+
+Samples and their augmentations are represented by different shades of the same color. The similarity of each sample's embedding with each of the prototypes (cluster-centers) is computed. The sample is then assigned to the prototype with the highest similarity. The aim of the network is to assign an image and its augmentation to the same prototype.
+
+**Can it lead to any trivial solution?**
+
+A trivial solution is possible where every sample is assigned to the same group/cluster/prototype. This would cause a collapsed representation for any given input.
+
+**How to avoid trivial solution?**
+1. **Equipartition constraint**
+   
+    <center>
+    <img src="{{site.baseurl}}/images/week10/10-1/equipartition.png" style="width: 350px; background-color:#DCDCDC;" /><br>
+    Figure 9: Clustering using an equipartition constraint
+    </center>
+
+    Given n samples and k partitions, each cluster is allowed to have a maximum of n/k samples. Embeddings are equally partitioned and this prevents the single cluster trivial solution. Optimal-transport based clustering methods such as the Sinkhorn-Knopp algorithm inherently guarantees the constraint.
+
+
+2. **Soft assignment**
+   
+    <center>
+    <img src="{{site.baseurl}}/images/week10/10-1/soft-assignment.png" style="width: 450px; background-color:#DCDCDC;" /><br>
+    Figure 10: Clustering using soft assignment
+    </center>
+    In soft assignment, a sample belongs to all prototypes with some proportion of each, such that they all sum up to 1.0. The composition values can be treated as a code that indicates how each embedding is encoded in the 'prototype' space.
+
+**Consider the case where we have a high class imbalance. If we use the equipartition method, then won't that give inacurate results?**
+
+Soft assignment solves this problem by representing a lot more classes (logically) than hard assignment. It gives a richer representation and is less sensitive to $k$ (number of classes) and hence $k/n$. However, if hard assignment is used, the class imbalance and the fixed value of $n/k$ will create inaccuracies.
+
+#### Training SwAV:
+
+<center>
+<img src="{{site.baseurl}}/images/week10/10-1/swav.png" style="width: 600px; background-color:#DCDCDC;" /><br>
+Figure 11: Architecture of SwAV
+</center>
+
+2 crops from an image are passed through the network to compute the corresponding codes. The $2^{nd}$ code is then predicted from the $1^{st}$ embedding and vice versa. This task forces the network to become invariant to data augmentations. Gradients are backpropagated onto the prototypes as well. Hence this model is updated in an online manner.
```

Comment:
`nd` is the product of `n` and `d`. Use `\text{}` if necessary. Here and everywhere else.